sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -538,7 +539,10 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
    * @since 2.0.0
    */
   @scala.annotation.varargs
-  def csv(paths: String*): DataFrame = format("csv").load(paths : _*)
+  def csv(paths: String*): DataFrame = {
```

Comment:
Does CSV/JSON work if users explicitly specify the schema? That's more a production usecase

Reply:
same question here.

Reply:
Updated, currently supported, and `corresponding UT` has been added to each type of `***Suite`.